stablehlo/dialect/TypeInference.cpp

```diff
@@ -679,27 +688,29 @@ LogicalResult verifyReduceWindowOpInputsAndInferWindow(
           " and input: ", inputType, " with rank = ", inputType.getRank(), ".");
   }
 
-  // P3.
-  auto paddingOrErr = convertPaddingAttribute(padding, location);
-  if (failed(paddingOrErr)) return failure();
-
+  // reduce_window_i4
   auto windowStridesOrErr =
       convert1DAttribute(windowStrides, location, "window_strides");
   if (failed(windowStridesOrErr)) return failure();
+  // reduce_window_i5
   auto baseDilationsOrErr =
       convert1DAttribute(baseDilations, location, "base_dilations");
   if (failed(baseDilationsOrErr)) return failure();
+  // reduce_window_i6
   auto windowDilationsOrErr =
       convert1DAttribute(windowDilations, location, "window_dilations");
   if (failed(windowDilationsOrErr)) return failure();
-  auto windowReversalOrErr = convertWindowReversalAttribute(
-      windowReversal, location, "window_reversal");
-  if (failed(windowReversalOrErr)) return failure();
+  // reduce_window_c12, reduce_window_i7
+  auto paddingOrErr = convertPaddingAttribute(padding, location);
+  if (failed(paddingOrErr)) return failure();
 
+  // reduce_window_c5, reduce_window_c6, reduce_window_c7, reduce_window_c8,
+  // reduce_window_c9, reduce_window_c10, reduce_window_c11, reduce_window_c12
```

Comment:
Can we perhaps simplify this to something along the lines of "reduce_window_c5 ... reduce_window_c12"?

Reply:
The purpose of explicitly stating them was to make the constraints easier to find, but condensing them in the caller and expanding them in the callee doesn't lose this information.

Reply:
Added #1474 to update the checklist.

Reply:
I don't think this specific comment has been updated.

Reply:
It should be updated now.